CHANGELOG.md

```diff
@@ -7,7 +7,9 @@
     `~/.gradle/glean`. Therefore `clean` can be used without reinstalling
     Miniconda afterward every time.
 * Python:
-  * The timing distribution and memory distribution metric types are now supported in Python ([#762](https://github.com/mozilla/glean/pull/762), [#763](https://github.com/mozilla/glean/pull/763)) 
+  * **Breaking Change**: The `glean.util` and `glean.hardware` modules, which
```

Comment:
I assume this is coming from #764, and thus once that is merged this PR should apply cleanly?